htdocs/takepos/invoice.php

```diff
@@ -937,7 +937,7 @@ function fail($message)
 
 				$pu_ht = price2num($number / (1 + ((float) $vatratecleaned / 100)), 'MU');
```

Comment:
@eldy I think that this expression is not exactly what is needed - $number is interpreted as a number in the expression here - I suppose it should be treated by price2num first - I casted it for now .

Reply:
You're right
$number must be replaced into this line with 
price2num($number, 'MU');

Reply:
I changed it.